projects/observability/src/shared/components/iframe/iframe.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+
+@Component({
+  selector: 'ht-iframe',
+  template: `<iframe width="100%" height="100%" frameBorder="0" [src]="urlSafe"></iframe>`,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./iframe.component.scss']
+})
+export class IFrameComponent implements OnInit {
+  @Input() public source: string = '';
```

Comment:
The extra layer of component here isn't really necessary -  this is not a component that's useful outside a dashboard context, so we can just put an iframe element directly in the widget renderer.

Rest of comments are just general ones on the component, but likely don't apply if removing it.